lib/std/zig/render.zig

```diff
@@ -1505,16 +1505,18 @@ fn renderSwitchCase(
     const node_tags = tree.nodes.items(.tag);
     const token_tags = tree.tokens.items(.tag);
     const trailing_comma = token_tags[switch_case.ast.arrow_token - 1] == .comma;
+    const has_comment = blk: {
```

Comment:
I think we should make this variable name more descriptive to avoid future misuse.
```suggestion
    const has_comment_before_arrow = blk: {
```

Reply:
Agreed! Resolved in latest commit. Thanks for the review!